miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/MenuController.java

```diff
@@ -229,21 +229,13 @@ public void setSecurityManager(com.eaglegenomics.simlims.core.manager.SecurityMa
   }
 
   @RequestMapping("/mainMenu")
-  public ModelAndView mainMenu(ModelMap model) {
-    try {
-      User user = securityManager.getUserByLoginName(SecurityContextHolder.getContext().getAuthentication().getName());
-      model.put("title", "Home");
-      ObjectMapper mapper = new ObjectMapper();
-      model.put("favouriteWorkflows",
-          user.getFavouriteWorkflows().stream().map(Dtos::asDto).map(dto -> mapper.valueToTree(dto)).collect(Collectors.toList()));
-      if (Arrays.asList(user.getRoles()).contains("ROLE_EXTERNAL") && !Arrays.asList(user.getRoles()).contains("ROLE_INTERNAL")) {
-        return new ModelAndView("/pages/external/externalMain.jsp", model);
-      } else {
-        return new ModelAndView("/pages/mainMenu.jsp", model);
-      }
-    } catch (IOException e) {
-      return new ModelAndView("/login.jsp", model);
-    }
+  public ModelAndView mainMenu(ModelMap model) throws IOException {
+    User user = securityManager.getUserByLoginName(SecurityContextHolder.getContext().getAuthentication().getName());
+    model.put("title", "Home");
+    ObjectMapper mapper = new ObjectMapper();
+    model.put("favouriteWorkflows",
+        user.getFavouriteWorkflows().stream().map(Dtos::asDto).map(dto -> mapper.valueToTree(dto)).collect(Collectors.toList()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)